### PR TITLE
Update dependencies for node 18.13.0

### PR DESCRIPTION
**Pull request by CGastrell**
Janitorial: this PR updates packages and node version to 18.13.0

Includes an update to the GH action to generate builds.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,26 +3,22 @@
...
-install: install-node install-php
+install: install-pnpm install-node install-php
 
 # Install Node dependencies
 install-node:
-	npm install
+	pnpm install
+
+install-pnpm:
+	npm install -g pnpm@7.27.1
```
Comment (ice9js) [inline Makefile:13]: This is the only thing I'd say is a strong blocker. I'd much rather have the extra line of instructions in our readme or something than having `make` implicitly change my global environment without any sort of warning!

**Review by ice9js — CHANGES_REQUESTED** [review]
All good, minus the `install-pnpm` step on the `Makefile`. I'd say let's just drop it to be honest.

File: Makefile (hunk 1 above)
Comment (CGastrell) [reply Makefile:13]: Mmm... fair enough

Comment (CGastrell) [reply Makefile:13]: It's on the development-environment.md, I'll remove it from the Makefile though

**Review by ice9js — APPROVED** [review]
Thanks Christian!

**Pull request merged**